pkg/module/sync/runner.go

```diff
@@ -2,8 +2,270 @@ package sync
 
 import (
 	"context"
+	"errors"
+	"fmt"
+	"time"
 
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	machineryruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	"github.com/kyma-project/lifecycle-manager/pkg/channel"
+	commonerrs "github.com/kyma-project/lifecycle-manager/pkg/common" //nolint:importas // a one-time reference for the package
+	"github.com/kyma-project/lifecycle-manager/pkg/log"
+	"github.com/kyma-project/lifecycle-manager/pkg/module/common"
+	"github.com/kyma-project/lifecycle-manager/pkg/util"
 )
 
+func New(clnt client.Client) *Runner {
+	return &Runner{
+		Client:    clnt,
+		versioner: schema.GroupVersions(clnt.Scheme().PreferredVersionAllGroups()),
+		converter: clnt.Scheme(),
+	}
+}
+
+type ModuleMetrics interface {
+	RemoveModuleStateMetrics(kymaName, moduleName string)
+}
+
 type GetModuleFunc func(ctx context.Context, module client.Object) error
+
+type Runner struct {
+	client.Client
+	versioner machineryruntime.GroupVersioner
+	converter machineryruntime.ObjectConvertor
+}
+
+// ReconcileManifests implements Runner.Sync.
```

Comment:
I don't get this comment. Can you explain if you know? If not, let's remove it.

Reply:
it's all coming from old code, will remove it.